### DISCUSSION TITLE
Is there a way to compress a csv file sent over PlainTextResponse

### DISCUSSION THREAD

**Discussion by nicolabortignon** [fastapi/fastapi#7547] [Questions] [answered]
I was wondering if PlainTextResponse has some of the http compression turned on by default. (I get a 2.2MB file displayed in chrome inspector).
When I try to gzip the file it goes down to 750kb. 
So I was wondering if something can be done there

**Comment by phy25** [comment, marked as answer]
If you follow the issue template and search you should be able to find https://fastapi.tiangolo.com/advanced/middleware/#gzipmiddleware

**Comment by nicolabortignon** [comment]
Perfect, that worked, not sure my search of gzip didn't return anything.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue @nicolabortignon  :+1:
